app.py

```diff
@@ -42,32 +46,34 @@ def render_template_nocache(template_name, **args):
 
 @app.route("/")
 def index():
-    sql = """
-select
-    (select count from voter_status_counters_35209
-     where "Application Status" = 'A' and "Ballot Status" = 'A')
-        as returned_general,
-    (select count from voter_status_counters_35211
-     where "Application Status" = 'A' and "Ballot Status" = 'A')
-        as returned_special,
-    (select count from voter_status_counters_35211
-     where "Application Status" = 'A' and "Ballot Status" = 'total')
-        as applied_special,
-    (select file_update_time from updated_times
-     where election = '35211' order by job_time desc limit 1)
-        as update_time"""
-    stats = db.engine.execute(sql).first()
-
-    resp = make_response(render_template("index.html", stats=stats))
-    resp.headers.set("Cache-Control", "public, max-age=7200")
+    election = datetime(2021, 1, 5, tzinfo=pytz.timezone("US/Eastern"))
+    days_until_election = math.ceil(
+        (election - datetime.now(tz=pytz.timezone("US/Eastern"))).total_seconds()
+        / (60 * 60 * 24)
+    )
+
+    data = statewide_by_day(days_until_election)
+
+    # Use yesterday's data, not today's -- we don't have today's data yet
```

Comment:
FYI, it is possible to actually get the date of the most recent data update using the `updated_times` table, e.g.:
`select max(file_update_time) from updated_times;`

maybe worth using that?